src/sgame/sg_utils.cpp

```diff
@@ -945,3 +945,31 @@ glm::vec3 G_CalcMuzzlePoint( const gentity_t *self, const glm::vec3 &forward )
 	// Meh. I doubt it saves much. Casting to short ints might have, though. (copypaste)
 	return glm::floor( muzzlePoint + 0.5f );
 }
+
+//Dunno which ones are really useful, CBSE is not even self-contained.
+#include "sg_local.h"
+#include "Entities.h"
+#include "CBSE.h"
+#include "sg_cm_world.h"
+
+bool gentity_t::Damage( float amount, gentity_t* source,
```

Comment:
Entities.cpp might be a better place for this. You could have `Entities::Damage` similarly to `Entities::Kill` etc. (optional)

Reply:
I'll be honest: I do not want to touch the CBSE more than strictly required. The patch currently works. It took me a full day of dev (more, if one considers a normal day of work). I don't want to rework it a lot just for that over-engineered thing that initially introduced the regression by being merged unfinished.

Reply:
You don't have to touch CBSE any more that. It's just renaming the function basically

Reply:
Well, you said "optional", and I'll take it. I'll let people interested at this kind of nit-picking do the meaningless change themselves.